DataCollector/DoctrineDataCollector.php

```diff
@@ -222,15 +222,26 @@ public function getGroupedQueries()
             });
             $groupedQueries[$connection] = $connectionGroupedQueries;
         }
+
         foreach ($groupedQueries as $connection => $queries) {
             foreach ($queries as $i => $query) {
-                $groupedQueries[$connection][$i]['executionPercent'] = $query['executionMS'] / $totalExecutionMS * 100;
+                $groupedQueries[$connection][$i]['executionPercent'] =
+                    $this->executionTimePercentage($query['executionMS'], $totalExecutionMS );
```

Comment:
@stof the first parens is after 120 chars, so I think this was the right choice. @mikeSimonson you should remove the space after `$totalExecutionMS`.